### DISCUSSION TITLE
Enabling custom PGNs with NGT-1

### DISCUSSION THREAD

**Discussion by cedschmidt-lab** [canboat/canboatjs#180] [Q&A]
Hello everyone,

I'm working on a new project with SignalK and an Actisense NGT-1 device to compute wind and performance data in a N2K network. 
I installed the BandG performance plugin [link](https://github.com/htool/signalk-bandg-performance-plugin) which uses PGN 130824 to be sent by NGT-1. I understood the PGN should first be enabled in the NGT-1 Tx list but I have no idea how to do it. The Actisense SDK reference provides C API to do it but I don't know haow to call it from JS. Can someone help me with this?

Many thanks...

**Comment by sbender9** [comment]
I have not figured out how to do this. I have some code here that I wrote by reverse engineering the their windows app, but it doesn't seem to work well.
